diff -Nurp hsakmt-1.0.0-org/hsakmt/events.c hsakmt-1.0.0/hsakmt/events.c
--- hsakmt-1.0.0-org/hsakmt/events.c	2015-10-04 10:59:41.000000000 +0300
+++ hsakmt-1.0.0/hsakmt/events.c	2015-11-13 19:21:24.043780978 +0200
@@ -228,6 +228,7 @@ hsaKmtWaitOnMultipleEvents(
 	HSAKMT_STATUS result;
 	struct kfd_event_data *event_data;
 	struct kfd_ioctl_wait_events_args args;
+	HSAuint32 i;
 
 	CHECK_KFD_OPEN();
 
@@ -235,7 +236,7 @@ hsaKmtWaitOnMultipleEvents(
 		return HSAKMT_STATUS_INVALID_HANDLE;
 
 	event_data = malloc(NumEvents * sizeof(struct kfd_event_data));
-	for (HSAuint32 i = 0; i < NumEvents; i++) {
+	for (i = 0; i < NumEvents; i++) {
 		event_data[i].event_id = Events[i]->EventId;
 		event_data[i].kfd_event_data_ext = (uint64_t)(uintptr_t)NULL;
 	}
@@ -255,7 +256,7 @@ hsaKmtWaitOnMultipleEvents(
 	}
 	else {
 		result = HSAKMT_STATUS_SUCCESS;
-		for (HSAuint32 i = 0; i < NumEvents; i++) {
+		for (i = 0; i < NumEvents; i++) {
 			if (Events[i]->EventData.EventType == HSA_EVENTTYPE_MEMORY) {
 				Events[i]->EventData.EventData.MemoryAccessFault.VirtualAddress = event_data[i].memory_exception_data.va;
 				result = gpuid_to_nodeid(event_data[i].memory_exception_data.gpu_id, &Events[i]->EventData.EventData.MemoryAccessFault.NodeId);
